src/ripple/consensus/Validations.h

```diff
@@ -192,6 +222,7 @@ class Validations
         decay_result_t<decltype (&Validation::ledgerID)(Validation)>;
     using NodeKey = decay_result_t<decltype (&Validation::key)(Validation)>;
     using NodeID = decay_result_t<decltype (&Validation::nodeID)(Validation)>;
```

Comment:
RCLConsensus.h is using `NodeID_t` instead of `NodeID`.  Is it worth making that consistent?  So, for example `SeqType` would be `Seq_t`?  Just for your consideration.  Personally, I really like it when a given type retains the same name for most or all of the code base.  That makes it way easier to grep for instances.  It's nice when it works that way, but it's hard to enforce.